### ISSUE TITLE
Crash on joining world with most recent version

### ISSUE THREAD

**Issue by Neposis** [PssbleTrngle/CreateJetpack#56]
### What happened?

With the new version installed, joining the world crashes the game (I also deleted my settings file so it could generate the correct key)

### Mod Version

3.0.1

### Minecraft Version

None

### Forge Version

43.2.6

### Relevant log output

```shell
[21:41:40] [Render thread/ERROR]: Exception caught during firing event: lateinit property binding has not been initialized
	Index: 5
	Listeners:
		0: HIGH
		1: ASM: class dev.architectury.event.forge.EventHandlerImplCommon event(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		2: NORMAL
		3: ASM: class com.endertech.minecraft.forge.world.GameWorld$WorldData onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		4: ASM: class com.legacy.structure_gel.core.events.SGClientEvents$ForgeBus clientPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		5: net.minecraftforge.eventbus.EventBus$$Lambda$4342/0x0000000800d89338@5fb0616d
		6: ASM: me.paulf.fairylights.server.ServerEventHandler@78c4f0e6 onTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		7: net.minecraftforge.eventbus.EventBus$$Lambda$4342/0x0000000800d89338@20adb33
		8: ASM: INSTANCE playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		9: ASM: class com.mrcrayfish.guns.common.ReloadTracker onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		10: ASM: class com.mrcrayfish.guns.client.render.gun.ModelOverrides onClientPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		11: ASM: class com.buuz135.industrial.item.infinity.OneThreeFiveHandler onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		12: ASM: class shadows.placebo.patreon.PatreonPreview tick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		13: ASM: se.mickelus.tetra.effect.ItemEffectHandler@28d2fb61 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		14: ASM: alpvax.mc.goprone.GoProne@fabad69 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		15: ASM: me.desht.pneumaticcraft.common.event.PneumaticArmorHandler@73bf8b9e onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		16: ASM: class me.desht.pneumaticcraft.common.pneumatic_armor.CommonArmorHandler$Listeners onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		17: net.minecraftforge.eventbus.EventBus$$Lambda$4342/0x0000000800d89338@457fe4be
		18: ASM: stepsword.mahoutsukai.handlers.ServerHandler@388c7b42 tickCheck(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		19: ASM: class com.teamabnormals.upgrade_aquatic.core.events.EntityEvents onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		20: ASM: class com.stal111.forbidden_arcanus.event.TickListener onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		21: ASM: class com.stal111.forbidden_arcanus.common.event.PlayerEvents onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		22: ASM: class com.hollingsworth.arsnouveau.common.potions.GravityEffect entityTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		23: ASM: class com.stal111.forbidden_arcanus.client.event.ClientPlayerEvents onClientPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		24: ASM: class com.hollingsworth.arsnouveau.common.event.ScryEvents playerTickEvent(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		25: ASM: class com.hollingsworth.arsnouveau.common.event.ManaCapEvents onTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		26: ASM: class com.hollingsworth.arsnouveau.common.event.ManaCapEvents playerOnTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		27: ASM: class com.hollingsworth.arsnouveau.common.event.LightEvents onTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		28: ASM: class net.mehvahdjukaar.supplementaries.common.events.forge.ServerEventsForge onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		29: ASM: vazkii.quark.content.building.module.HollowLogsModule@52671757 playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		30: ASM: vazkii.quark.content.management.module.AutomaticToolRestockModule@2d94d321 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		31: ASM: vazkii.quark.content.tools.module.PathfinderMapsModule@210eca0e playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		32: ASM: vazkii.quark.content.tools.module.ColorRunesModule@2a309139 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		33: ASM: vazkii.quark.content.tweaks.module.GrabChickensModule@598d97f6 playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		34: ASM: vazkii.quark.content.tweaks.module.EnhancedLaddersModule@1974e0fd onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		35: ASM: vazkii.quark.content.tweaks.module.CompassesWorkEverywhereModule@3ef0f59 onUpdate(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		36: ASM: vazkii.quark.content.tweaks.module.CampfiresBoostElytraModule@474cacfe onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		37: ASM: class com.simibubi.create.content.curiosities.bell.HauntedBellPulser hauntedBellCreatesPulse(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		38: ASM: class com.simibubi.create.content.contraptions.components.structureMovement.ContraptionHandlerClient preventRemotePlayersWalkingAnimations(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		39: net.minecraftforge.eventbus.EventBus$$Lambda$4342/0x0000000800d89338@54edb96d
		40: net.minecraftforge.eventbus.EventBus$$Lambda$4342/0x0000000800d89338@88a1194
		41: net.minecraftforge.eventbus.EventBus$$Lambda$4342/0x0000000800d89338@5c0d80f5
		42: ASM: mekanism.common.CommonPlayerTickHandler@4f3df021 onTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		43: ASM: com.mrcrayfish.guns.common.BoundingBoxManager@27b54c97 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		44: ASM: com.illusivesoulworks.comforts.client.ComfortsClientEventsListener@20faa9b5 onTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		45: ASM: com.mrcrayfish.guns.client.handler.AimingHandler@67ae1d3f onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		46: ASM: com.natamus.weakerspiderwebs.forge.events.ForgeWebEvent@51770295 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		47: net.minecraftforge.eventbus.EventBus$$Lambda$4342/0x0000000800d89338@666dc5de
		48: LOW
		49: ASM: se.mickelus.tetra.items.forged.VibrationDebuffer@3235c3fd onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		50: ASM: se.mickelus.tetra.items.modular.impl.toolbelt.booster.TickHandlerBooster@5391d3e4 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		51: LOWEST
		52: ASM: class xyz.przemyk.simpleplanes.client.ClientEventHandler onClientPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		53: ASM: class cofh.lib.util.ItemTracker playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		54: ASM: class com.hollingsworth.arsnouveau.common.event.EventHandler onGlideTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
kotlin.UninitializedPropertyAccessException: lateinit property binding has not been initialized
	at TRANSFORMER/flightlib@1.0.1/com.possible_triangle.flightlib.api.FlightKey.getBinding(FlightKey.kt:17)
	at TRANSFORMER/flightlib@1.0.1/com.possible_triangle.flightlib.logic.ControlSender.onTick(ControlSender.kt:41)
	at TRANSFORMER/flightlib@1.0.1/com.possible_triangle.flightlib.forge.ForgeEntrypoint.lambda$new$3(ForgeEntrypoint.java:44)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:315)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:296)
	at TRANSFORMER/forge@43.2.6/net.minecraftforge.event.ForgeEventFactory.onPlayerPreTick(ForgeEventFactory.java:805)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.world.entity.player.Player.m_8119_(Player.java:216)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.player.LocalPlayer.m_8119_(LocalPlayer.java:200)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.multiplayer.ClientLevel.m_104639_(ClientLevel.java:269)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.world.level.Level.m_46653_(Level.java:457)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.multiplayer.ClientLevel.m_194182_(ClientLevel.java:251)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.world.level.entity.EntityTickList.m_156910_(EntityTickList.java:54)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.multiplayer.ClientLevel.m_104804_(ClientLevel.java:249)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91398_(Minecraft.java:1791)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1078)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91374_(Minecraft.java:700)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.main.Main.m_239872_(Main.java:212)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.main.Main.main(Main.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at MC-BOOTSTRAP/fmlloader@1.19.2-43.2.6/net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)



Followed by:
[21:41:55] [Render thread/ERROR]: Reported exception thrown!
net.minecraft.ReportedException: Ticking entity
	at net.minecraft.world.level.Level.m_46653_(Level.java:466) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.client.multiplayer.ClientLevel.m_194182_(ClientLevel.java:251) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.world.level.entity.EntityTickList.m_156910_(EntityTickList.java:54) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.client.multiplayer.ClientLevel.m_104804_(ClientLevel.java:249) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.client.Minecraft.m_91398_(Minecraft.java:1791) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1078) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:700) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.client.main.Main.m_239872_(Main.java:212) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19.2-43.2.6.jar%23101!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) [modlauncher-10.0.8.jar%2388!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-10.0.8.jar%2388!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-10.0.8.jar%2388!/:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-10.0.8.jar%2388!/:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-10.0.8.jar%2388!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-10.0.8.jar%2388!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-10.0.8.jar%2388!/:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) [bootstraplauncher-1.1.2.jar:?]
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property binding has not been initialized
	at com.possible_triangle.flightlib.api.FlightKey.getBinding(FlightKey.kt:17) ~[flightlib-forge-1.0.1.jar%23740!/:1.0.1]
	at com.possible_triangle.flightlib.logic.ControlSender.onTick(ControlSender.kt:41) ~[flightlib-forge-1.0.1.jar%23740!/:1.0.1]
	at com.possible_triangle.flightlib.forge.ForgeEntrypoint.lambda$new$3(ForgeEntrypoint.java:44) ~[flightlib-forge-1.0.1.jar%23740!/:1.0.1]
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260) ~[eventbus-6.0.3.jar%2385!/:?]
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252) ~[eventbus-6.0.3.jar%2385!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.3.jar%2385!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.3.jar%2385!/:?]
	at net.minecraftforge.event.ForgeEventFactory.onPlayerPreTick(ForgeEventFactory.java:805) ~[forge-1.19.2-43.2.6-universal.jar%23716!/:?]
	at net.minecraft.world.entity.player.Player.m_8119_(Player.java:216) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.client.player.LocalPlayer.m_8119_(LocalPlayer.java:200) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.client.multiplayer.ClientLevel.m_104639_(ClientLevel.java:269) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	at net.minecraft.world.level.Level.m_46653_(Level.java:457) ~[client-1.19.2-20220805.130853-srg.jar%23711!/:?]
	... 21 more
```

**Comment by NamelessGamer06** [comment]
Also having the same problem.

**Comment by PssbleTrngle** [comment]
Will look into it, for now just switch the previous version since there where almost only internal changes

**Comment by PssbleTrngle** [comment]
Should be fixed in the lastest version

**Comment by Hyperbuss** [comment]
I'm having this same issue, trying to join a server and I'm considerd a ticking entity, then kicked. I'm using the latest version that's for Create 0.5.0i, that being 3.0.2 . Perhaps it's fixed in 3.1.0 but that one's only compatible with the lastes version of Creaate (0.5.0a), which isnt supported by enough other mods at the moment so mypack can't use. 

I understand if it's not worth packporting this fix for this but it would be appreciated by ~10 people I know :)

here's the log: https://mclo.gs/ArxBLhB

**Comment by mlus-asuka** [comment]
> 



> I'm having this same issue, trying to join a server and I'm considerd a ticking entity, then kicked. I'm using the latest version that's for Create 0.5.0i, that being 3.0.2 . Perhaps it's fixed in 3.1.0 but that one's only compatible with the lastes version of Creaate (0.5.0a), which isnt supported by enough other mods at the moment so mypack can't use.
> 
> I understand if it's not worth packporting this fix for this but it would be appreciated by ~10 people I know :)
> 
> here's the log: https://mclo.gs/ArxBLhB

same issue in 3.1.0.it tried to run client only class in server so it crashed.

**Comment by PssbleTrngle** [comment]
I think @Hyperbus02 issue is actually a different one than the original of this thread & a duplicate of #62, which should be resolved in version 3.1.1, but as mentioned in another issue, I will not backport this fix to a previous version of create
